resources/views/service/partials/addTraderUser.blade.php

```diff
@@ -0,0 +1,107 @@
+<div class="horizontal-container">
+    <div>
+        <label for="new-user-name"
+               class="required"
+        >User Name</label>
+        <input type="text"
+               id="new-user-name"
+               value=""
+               class=""
```

Comment:
do we need the class and value properties with empty strings for values? same on lines 19 and 20 below

Reply:
hmm. no